tools/infrasys/src/s3.rs

```diff
@@ -90,21 +89,24 @@ pub async fn create_s3_bucket(region: &str, stack_name: &str) -> Result<(String,
 /// Adds a BucketPolicy allowing GetObject access to a specified VPC
 /// Input: Region, Name of bucket, which prefix root.json should be put under, and vpcid
 /// Note that the prefix parameter must have the format "/<folder>/*" and the bucket name "<name>"
-/// Output: Doesn't need to save any metadata from this action  
+/// Output: Doesn't need to save any metadata from this action
 pub async fn add_bucket_policy(
     region: &str,
     bucket_name: &str,
     prefix: &str,
     vpcid: &str,
 ) -> Result<()> {
     // Get old policy
-    let s3_client =
-        S3Client::new(Region::from_str(region).context(error::ParseRegionSnafu { what: region })?);
+    let config = aws_config::from_env()
+        .region(Region::new(region.to_owned()))
```

Comment:
Maybe turn this into a `Region` or even an `SdkConfig` when `ValidRepoInfo` is created, so we don't need to do the conversion each time.

Reply:
Added Region, since the string will still be needed in some places. Not sure about SDKconfig, but, I'll have that conversation in more depth in @ecpullen's comment